hms-app/src/actions/IdeaActions.tsx

```diff
@@ -64,3 +64,37 @@ export const createIdea = (
     })
     .catch((err) => console.log(err))
 }
+
+export const editIdea = (
+  ideaID: string,
+  idea: {
+    ownerId: string
+    hackathonId: string
+    title: string
+    description: string
+    problem: string
+    goal: string
+  },
+  skills: string[],
+  categories: string[]
```

Comment:
why are they separate? are required skill and the category a part of the idea? as well as the id itself too?

Reply:
because of the checkbox input type it would need an extra step to put them all in the idea object

Reply:
I would honestly argue that is ok.  Or I would say to not allow you to pass in an incomplete Idea.  When you make it like this you have added another concept of what an `idea` is.  You could instead call this `ideaStringData` or something but honestly thats not great either